README.md

```diff
@@ -74,75 +74,69 @@ The documentation is generated using [Jazzy](https://github.com/realm/jazzy).
 
 ### Dynamic Framework Installation [⤴](#table-of-contents)
 
-2. Add `Buy.framework` target as a depenency by navigating to:
-  - `Your Project` and (Select your target)
-  - `Build Phases`
-  - `Target Dependencies`
-  -  Add  `Buy.framework`
-3. Link `Buy.framework` by navigating to:
-  - `Your Project` and (Select your target)
-  - `Build Phases`
-  - `Link Binary With Libraries`
-  -  Add `Buy.framework`
-4. Ensure the framework is copied into the bundle by navigating to:
-  - `Your Project` and (Select your target)
-  - `Build Phases`
-  - Add `New Copy Files Phase`
-  - Select `Destination` dropdown and chose `Frameworks`
-  - Add `Buy.framework`
-5. Import into your project files using `import Buy`
- 
+2. Add `Buy.framework` target as a dependency:
```

Comment:
I don't think that was intentional. The numbering should probably start at `#1`.